### ISSUE TITLE
Add JUnit XML support

### ISSUE THREAD

**Issue by beatsbears** [reportunit/reportunit#48]
Specifically to support Newman, which is the command line runner for Postman.  This outputs JUnit XML.

I've attached an example, though I had to change to .txt so github would accept the attachment. 
[Junit.txt](https://github.com/reportunit/reportunit/files/119351/Junit.txt)

**Comment by carlosrfernandez** [comment]
I've forked this project and have started implementing the JUnit parser. 

I don't know a lot about JUnit, it will be helpful if you can provide additional JUnit results so that once I have the main logic implemented, I can properly test it. :wink:

**Comment by beatsbears** [comment]
Definitely! I Admittedly don't have much experience either. This is an example from Postman that should be similar to what I'll be working with. 
http://help.catchsoftware.com/display/ET/JUnit+Format seems to have a nice rundown of the specs. 

[newmanXML.txt](https://github.com/reportunit/reportunit/files/186010/newmanXML.txt)

**Comment by carlosrfernandez** [comment]
Thanks!

I've already implemented the validation of the input file using JUnit's official XSD schema.

I'll try to get a basic report up and running in the next couple of days.

**Comment by carlosrfernandez** [comment]
Have a look at https://github.com/carlosrfernandez/reportunit
I've got the basic report stuff done. 

There's a couple of things still needed though. 
- I still need to parse the `<properties>` tag that appear in some JUnit xml results that I've seen online, and figure out where to store that information, I don't want to alter the base model too much without first discussing it with the developers of this project. 
- It needs further testing for different types of files (ie: different tests with different projects attributes etc)

Give it a go and see if it works for you. I will try to get some more stuff and refactor some of the code that I wrote in the next few days.

**Comment by beatsbears** [comment]
Thank you so much!  I'll try to take a look at this in the next few days and provide any feedback I can.

**Comment by agray** [comment]
What's the current status of support for JUnit?

**Comment by carlosrfernandez** [comment]
jUnit's basic schema is supported ;)

**Comment by agray** [comment]
I see it in master branch, however was it included in NuGet package
1.5.0-beta1?

Regards,

Andrew

**Comment by abhiras29** [comment]
@carlosrfernandez : I tried to run your source code from https://github.com/carlosrfernandez/reportunit 
But it is not picking the Input Directory itself and throwing below error :- 

_**C:\POCs_Automation\reportunit-1.2>ReportUnit.exe C:\POCs_Automation\reportunit-1.2\CaseDesignerSettingsSuite_20171114-142301.xml C:\POCs_Automation\reportunit-1.2\Results.html

--
ReportUnit v1.5. Report generator for the test-runner family.
http://reportunit.relevantcodes.com/
Copyright (c) 2015 Anshoo Arora (Relevant Codes)
Developers:  Anshoo Arora, Sandra Greenhalgh
--


Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'RazorEngine, Version=3.10.0.0, Culture=neutral, PublicKeyToken=9ee697374c7e744a' or one of its dependencies. The system cannot find the file specified.
   at ReportUnit.ReportUnitService.CreateReport(String input, String outputDirectory)
   at ReportUnit.Program.Main(String[] args)**_
